QUARKUS-343.md

```diff
@@ -0,0 +1,52 @@
+# QQE-343- Quarkus and non JSON payload in REST endpoints
```

Comment:
MD file is named QUARKUS-343 while JIRA name is QQE-343, that's confusing.

Reply:
Rate on the scale 1-10 how much I irritate you :-D

Reply:
If it's ok to change the name to `QQE-343.md `I am ok with that but will be the first file with that kind of name...

Reply:
done!